ci/ReleaseTool/Common.cs

```diff
@@ -55,5 +57,108 @@ private static string AppendDirectorySeparator(string originalPath)
 
             return originalPath + directorySeparator;
         }
+
+        public static bool IsMarkdownHeading(string markdownLine, int level, string startTitle = null)
+        {
+            var heading = $"{new string('#', level)} {startTitle ?? string.Empty}";
+
+            return markdownLine.StartsWith(heading);
+        }
+
+        public static bool UpdateChangeLog(string ChangeLogFilePath, string Version, GitClient gitClient, string ChangeLogReleaseHeadingTemplate)
```

Comment:
Think these arg names are using the wrong case style for C#

Reply:
yes, probably, I moved quite a few functions into the Common file, since I needed to reuse them, and I also added some new ones by just pasting in the required variables from where they were previously used.
Will need a pass.